src/NzbDrone.Core/Download/Clients/Vuze/Vuze.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using FluentValidation.Results;
+using NLog;
+using NzbDrone.Common.Disk;
+using NzbDrone.Common.Http;
+using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Download.Clients.Transmission;
+using NzbDrone.Core.MediaFiles.TorrentInfo;
+using NzbDrone.Core.RemotePathMappings;
+
+namespace NzbDrone.Core.Download.Clients.Vuze
+{
+    public class Vuze : Transmission.Transmission
+    {
+        private const int MINIMAL_SUPPORTED_PROTOCOL_VERSION = 14;
+        private readonly ITransmissionProxy _proxy;
+
+        public Vuze(ITransmissionProxy proxy,
+                    ITorrentFileInfoReader torrentFileInfoReader,
+                    IHttpClient httpClient,
+                    IConfigService configService,
+                    IDiskProvider diskProvider,
+                    IRemotePathMappingService remotePathMappingService,
+                    Logger logger)
+            : base(proxy, torrentFileInfoReader, httpClient, configService, diskProvider, remotePathMappingService, logger)
+        {
+            _proxy = proxy;
+        }
+
+        protected override DownloadClientItem CreateDownloadItem(TransmissionTorrent torrent, OsPath outputPath)
```

Comment:
Since this method is exactly the same as the base method except for the path `GetOutputPath` would be a better method to override in the Vuze class. Vuze would return the path that its reporting and Transmission would return the path + torrent name. Less code moving and you don't need to alter a path only to recreate it.